tex/latex/biblatex/biblatex.sty

```diff
@@ -5143,6 +5143,9 @@
   \do{winter}%
   \do{am}%
   \do{pm}%
+  \do{filed}
```

Comment:
The lines here are missing `%`s at the end
```suggestion
  \do{filed}%
```
etc. etc. (apparently, I can't comment on multiple lines at once...)

Reply:
ok